app/background/events/cosmeticFilterEvents.ts

```diff
@@ -0,0 +1,116 @@
+// import windowActions from '../actions/cosmeticFilterActions'
+// import tabActions from '../actions/tabActions'
+
+// import * as cosmeticFilterAPI from '../api/cosmeticFilterAPI'
+// background
+
+// this definitely goes into state
+import cosmeticFilterActions from '../actions/cosmeticFilterActions'
+
+let rule = {
+  host: '',
+  selector: ''
+}
+
+// add context menu
+chrome.runtime.onInstalled.addListener(function () {
+
```

Comment:
nit: can we remove this empty line? standardjs would complain but I'm not sure why it doesn't